src/client/components/ListOfCertifications/index.js

```diff
@@ -10,7 +9,7 @@ import AccordionItem from "../../components/AccordionItem";
 function ListOfCertifications(props) {
   const { t } = useTranslation();
 
-  const { header, userData } = props;
```

Comment:
I moved the header outside of the component onto the parent page, because the Staff View version of the confirmation page uses h3 headers instead of h2 on the

Reply:
Never mind, I'm no longer using h3 headers, but I'm leaving the header outside of the component :)